RxASDataSources.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
      s.name = 'RxASDataSources'
-     s.version = '0.5.0'
+     s.version = '0.6.0'
```

Comment:
Should we bump version to 1.0 to follow version sematic also supporting new Texture version